shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1484,6 +1484,83 @@ public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOE
         }
     }
 
+    @Data
+    public static final class MergeSegmentsBatch implements Request, WireCommand {
```

Comment:
Please bump version number at the top of the file.

Reply:
Fixed